ra_aid/project_info.py

```
@@ -0,0 +1,123 @@
+        new_status = is_new_project(directory)
+        
+        # Get file listing
+        files, total = get_file_listing(directory, limit=file_limit)
+        
+        return ProjectInfo(
+            is_new=new_status,
+            files=files,
+            total_files=total
+        )
+        
+    except (ProjectStateError, FileListerError) as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)